### PR TITLE
Add pricing plans

### PR DESCRIPTION
**Pull request by daltonfury42**
image

### CODE REVIEW

**Comment by maaverik** [comment]
Don't you think 23$ is too much :joy:? Let's maybe say 10?

**Comment by daltonfury42** [comment]
It's for bigger companies. I'll update to $8.99

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 1]
```diff
@@ -127,11 +127,51 @@ export const ExtraInfo = () => (
...
 );
 
-export const GetStarted = () => (
+export const Pricing = () => (
   <div data-aos="fade-up" className={`${styles['section']} ${styles['get-started']}`}>
-    <h2 data-aos="zoom-in">Ready to get started with SimplQ?</h2>
-    <div className={`${styles['button-group']}`}>
-      <StandardButton onClick={smoothScrollToHomePageTop}>Start Now</StandardButton>
+    <h2 data-aos="zoom-in">Pricng - Plans for every usecase</h2>
+    <div className={styles['container']}>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:133]: Pricing is misspelled

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 2]
```diff
@@ -127,11 +127,51 @@ export const ExtraInfo = () => (
...
+      <div className={styles['price-card']}>
+        <h1>Free</h1>
+        <p className={styles['subtitle']}>Basic for small businesses</p>
+        <li>
+          <ul>Unlimeted Queues</ul>
+          <ul>6000 tokens per month</ul>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:139]: Unlimited

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 3]
```diff
@@ -127,11 +127,51 @@ export const ExtraInfo = () => (
...
+        </li>
+        <p className={styles['amount']}>$0</p>
+        <StandardButton onClick={smoothScrollToHomePageTop}>Start for free</StandardButton>
+      </div>
+      <div className={styles['price-card']}>
+        <h1>Business</h1>
+        <p className={styles['subtitle']}>For bigger companies</p>
+        <li>
+          <ul>Everything in free plan.</ul>
+          <ul>20,000 tokens per month.</ul>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:149]: Everything in the free plan

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 4]
```diff
@@ -127,11 +127,51 @@ export const ExtraInfo = () => (
...
+        <p className={styles['amount']}>$8.99/mo</p>
+        <StandardButton onClick={smoothScrollToHomePageTop}>Upgrade</StandardButton>
+      </div>
+      <div className={styles['price-card']}>
+        <h1>Enterprise</h1>
+        <p className={styles['subtitle']}>Best for unique requirements that need to scale.</p>
+        <li>
+          <ul>Everything in business plan.</ul>
+          <ul>No token or user limits.</ul>
+          <ul>In house installation support.</ul>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:163]: I think we shouldn't use the word token, it's hard to understand what that means. If we are using it, we should define somewhere that it means users added to the queue.

Comment (daltonfury42) [reply simplq/src/components/pages/Home/StaticInfos.jsx:163]: I think it's understandable that we are a queue management system and that everyone gets a token number when they join the queue. 

I also sort of want to get the word "token" in the page as I want us to come for "token system" search etc.

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 5]
```diff
@@ -127,11 +127,51 @@ export const ExtraInfo = () => (
...
+        </li>
+        <p className={styles['amount']}>$8.99/mo</p>
+        <StandardButton onClick={smoothScrollToHomePageTop}>Upgrade</StandardButton>
+      </div>
+      <div className={styles['price-card']}>
+        <h1>Enterprise</h1>
+        <p className={styles['subtitle']}>Best for unique requirements that need to scale.</p>
+        <li>
+          <ul>Everything in the business plan.</ul>
+          <ul>No token or user limits.</ul>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:162]: Only this point mentions users. I feel that we should either remove users here or add users to the other sections too.

Comment (daltonfury42) [reply simplq/src/components/pages/Home/StaticInfos.jsx:162]: I'll remove this point, it's not needed.

**Review by maaverik — APPROVED** [review]

**Pull request merged**